spec/acceptance/trusted_external_data_spec.rb

```diff
@@ -109,4 +110,21 @@
       expect(trusted_json['external']['servicenow']['fqdn']).to eql(master.uri)
     end
   end
+
+  context 'user specifies a hiera-eyaml encrypted oauth token' do
```

Comment:
Yeah, I guess just this test is enough to test that things work w/ an oauth token. I'd maybe add a comment explaining that in case someone else thinks of this as two separate things like me ("test oauth token works", "test hiera-eyaml encrypted oauth token works").

Reply:
Cool, thanks. What about https://github.com/puppetlabs/puppetlabs-servicenow_cmdb_integration/pull/25/files#r444433831 ?